.env.development

```diff
@@ -0,0 +1,4 @@
+NEXT_PUBLIC_ALGOLIA_APP_ID=XQEP3AD9ZT
+NEXT_PUBLIC_ALGOLIA_API_KEY=118b51a67eefbe4d3c445d1e1090afac
+NEXT_PUBLIC_SHOPIFY_DOMAIN=ifixit-test.myshopify.com
+NEXT_PUBLIC_SHOPIFY_STOREFRONT_ACCESS_TOKEN=bc6c5d0972b3ad5d565c99768e6065bf
```

Comment:
Are these safe to be public? This repo isn't private, so they're world-visible.

Reply:
They all should be accessible by the client in order to be able to make request to Algolia or Shopify storefront, so yes there should be no issue here.